queries/dune_v2/period_slippage.sql

```diff
@@ -7,24 +8,22 @@ filtered_trades as (
             -- Estimation made here: https://dune.com/queries/1646084
                 then ((gas_used - 73688 - (70528 * num_trades)) / 90000)::int
                 else dex_swaps
-            end as dex_swaps,
+           end as dex_swaps,
            num_trades,
            solver_address,
-           trader                                              as trader_in,
-           -- Null Receiver: https://dune.com/queries/1729130
-           -- TODO - This could also be fixed in the spellbook!
-           case
-                when receiver = '0x0000000000000000000000000000000000000000'
-                then trader
-            else receiver end                                  as trader_out,
```

Comment:
This was fixed between last week and now:

https://github.com/duneanalytics/spellbook/pull/2300